main/inc/lib/chat.lib.php

```diff
@@ -244,6 +244,26 @@ public function sanitize($text)
         $text = str_replace("\n", "<br>", $text);
         return $text;
     }
+    
+    public static function  set_disable_chat($status)
+    {
+            $_SESSION['disable_chat'] = $status;
+    }
+    
+    public function disable_chat()
```

Comment:
Missing documentation block (and should be static) but then the code using it should change to a static call as well.